Makefile

```
@@ -65,7 +65,46 @@ python_sdk::
 		rm ./bin/setup.py.bak && \
 		cd ./bin && python3 setup.py build sdist
 
+gen_examples::
+	cd ${WORKING_DIR}/examples/yaml && \
+	rm -rf ${WORKING_DIR}/examples/{go,nodejs,python,dotnet} && \
+	cd ${WORKING_DIR}/examples/yaml && pulumi convert --logtostderr --generate-only --non-interactive --language go --out ${WORKING_DIR}/examples/go 2>&1 && \
+	cd ${WORKING_DIR}/examples/yaml && pulumi convert --logtostderr --generate-only --non-interactive --language nodejs --out ${WORKING_DIR}/examples/nodejs 2>&1 && \
+	cd ${WORKING_DIR}/examples/yaml && pulumi convert --logtostderr --generate-only --non-interactive --language python --out ${WORKING_DIR}/examples/python 2>&1 && \
+	cd ${WORKING_DIR}/examples/yaml && pulumi convert --logtostderr --generate-only --non-interactive --language dotnet --out ${WORKING_DIR}/examples/dotnet 2>&1
+
+pulumi_login::
+	export PULUMI_CONFIG_PASSPHRASE="asdfqwerty1234" && \
+	pulumi login --local
```

We shouldn't need to repeat the password in multiple places in the `Makefile`.
```suggestion
local_password: export PULUMI_CONFIG_PASSPHRASE = asdfqwerty1234

pulumi_login: local_password
	pulumi login --local
```

---

Resolved with https://github.com/pulumi/pulumi-provider-boilerplate/pull/80/commits/cf48af22aa0e5913d3a5fa54c2b30d309df7abc9